lib/paymentrequest.py

```diff
@@ -366,6 +366,22 @@ def make_unsigned_request(req):
     pr.signature = util.to_bytes('')
     return pr
 
+def get_payment(paymentfile):
+    p = pb2.Payment()
+    try:
+      f = open(paymentfile, "rb")
+      p.ParseFromString(f.read())
+      f.close()
+    except IOError:
+      print (paymentfile + ": Could not open payment file")
```

Comment:
Can you please use `print_error` (it's defined in `util.py` and is imported as `print_error` in this file's namespace) -- I realize some of the code in this file uses print -- but it really should use `print_error` which logs errors properly.